ebpfapi/platform.cpp

```diff
@@ -39,19 +40,40 @@ CreateFileW(
     uint32_t flags_and_attributes,
     _In_opt_ ebpf_handle_t template_file)
 {
-    return ::CreateFileW(
+    return reinterpret_cast<ebpf_handle_t>(::CreateFileW(
         file_name,
         desired_access,
         share_mode,
         security_attributes,
         creation_disposition,
         flags_and_attributes,
-        template_file);
+        reinterpret_cast<HANDLE>(template_file)));
 }
 
 bool
 CloseHandle(_In_ _Post_ptr_invalid_ ebpf_handle_t handle)
 {
-    return ::CloseHandle(handle);
+    return ::CloseHandle(reinterpret_cast<HANDLE>(handle));
+}
+
+int
+_open_osfhandle(intptr_t os_file_handle, int flags)
+{
+    return ::_open_osfhandle(os_file_handle, flags);
+}
+
+intptr_t
+_get_osfhandle(int file_handle)
```

Comment:
Should `file_handle` be renamed to `file_descriptor`? as it confuses with the 64bit handle.